@types/ol-ext/overlay/Popup.d.ts

```diff
@@ -120,7 +124,7 @@ export default class Popup extends Overlay {
     popup.show("New informations");
     * @api stable
      */
-    show(coordinate: Coordinate | undefined, features: Feature | Feature[]): void;
+    show(coordinate: Coordinate | string, html: string | undefined): void;
```

Comment:
```suggestion
    show(coordinate: Coordinate | string, html?: string): void;
```
If you use the ? you can omit the undefined (it's just another syntax variant for the same)

Reply:
@Siedlerchr Okay, thanks for advice!
I will check the original code in `ol-ext`, and some of the code inside the constructor/methods seem to be outdated, so about this part, I will keep the original `undefined` to truck future update possibility.